CONTRIBUTING.md

```diff
@@ -77,13 +77,13 @@ If you only work on a single package you can watch only for that one by calling:
 
 ```sh
 # e.g. `$ npm run watch:pkg @wdio/runner`
-$ npm run watch:pkg <package-name>
+$ npm run watch <package-name>
 ```
 
 It is also a good idea to run jest in watch mode while developing on a single package to see if changes affect any tests:
 
 ```sh
-$ npx jest ./packages/<package-name>/tests --watch
+$ npx jest ./packages/<package-name>/tests --watch --coverageReporters lcov
```

Comment:
define the coverageReporters in the config file instead?

Reply:
We do. This just changes the coverage reporter so you don't get to see this coverage list anymore and instead see the test results.